### DISCUSSION TITLE
How to manually set column style color-mapping for string columns?

### DISCUSSION THREAD

**Discussion by henryli78** [perspective-dev/perspective#2000] [Q&A]
Hi everyone,

When working with a string column in Perspective, I want to use the Color: Series functionality in Perspective but specify a color scheme dependent on the values within my string column. 

For example, I know that the values of my string column can take on either `"Apple"` or `"Orange"`. I want to ensure that `"Apple"` is always red and `"Orange"` is always orange. Every time I restart my program, it seems like Perspective chooses a new coloring scheme for `"Apple"` and `"Orange"` (the number of rows of each do change each time which may be related). 

Is pre-defining some color mapping for certain string values possible in Perspective? 

This is the column style I am talking about: 
![image](https://user-images.githubusercontent.com/49212028/199037430-b9c606e9-8d00-4e0e-9725-50921d41c16b.png)

**Comment by kzmKZuzmo** [comment]
@texodus  can you help with this ? how can we define color for specific value ?

**Comment by texodus** [comment]
Perspective does not support this currently. You can define a new [theme](https://perspective.finos.org/guide/how_to/javascript/custom_themes.html) if you want to change the default color palette.
